src/components/TitleItem.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <h1>
+    {{ props.title }}
+  </h1>
+</template>
+
+<script setup lang="ts">
+  const props = defineProps(["title"])
```

Comment:
If you are not planning to use your props in the `script` section then you don't have to assign them to a variable, since they are gonna be exposed automatically.